CHANGELOG.rst

```diff
@@ -21,6 +21,8 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 * Constructor using the deprecated ``life.qbic.datamodel.dtos.business.ProductId`` constructor for ``life.qbic.datamodel.dtos.business.services.Sequencing``, ``life.qbic.datamodel.dtos.business.services.SecondaryAnalysis`,
 ``life.qbic.datamodel.dtos.business.services.ProteomicAnalysis``, ``life.qbic.datamodel.dtos.business.services.ProjectManagement``, ``life.qbic.datamodel.dtos.business.services.PrimaryAnalysis`,
 ``life.qbic.datamodel.dtos.business.services.MetabolomicAnalysis``, ``life.qbic.datamodel.dtos.business.services.DataStorage``
+* Override ``equals()`` method for ``life.qbic.datamodel.dtos.business.OfferId` and ``life.qbic
+.datamodel.dtos.business.TomatoId``properly
```

Comment:
Nice addition to the Changelog. I think this belongs into the Fixed section and not the Deprecated section though.